docs/connectors/sources/csv-source.md

```diff
@@ -0,0 +1,49 @@
+# CSV Source
+
+A basic source that reads data from a single CSV file.
```

Comment:
It would be nice also to mention that the file is processed only once when the source starts.
I.e., it doesn't watch the file for new updates, and on every restart it will produce new messages to the topic.